README.md

```diff
@@ -1,23 +1,33 @@
+[![Go Version](https://img.shields.io/badge/go-v1.19-green.svg)](https://golang.org/dl/)
 [![Lint and Test](https://github.com/gballet/go-verkle/actions/workflows/go.yml/badge.svg)](https://github.com/gballet/go-verkle/actions/workflows/go.yml)
 [![DeepSource](https://deepsource.io/gh/gballet/go-verkle.svg/?label=active+issues&show_trend=true&token=OjuF5Q2HbKzpWY8LgWuffNZp)](https://deepsource.io/gh/gballet/go-verkle/?ref=repository-badge)
 [![goreports](https://goreportcard.com/badge/github.com/gballet/go-verkle)](https://goreportcard.com/report/github.com/gballet/go-verkle)
 [![API Reference](https://camo.githubusercontent.com/915b7be44ada53c290eb157634330494ebe3e30a/68747470733a2f2f676f646f632e6f72672f6769746875622e636f6d2f676f6c616e672f6764646f3f7374617475732e737667)](https://pkg.go.dev/github.com/gballet/go-verkle)
 [![Block replay](https://github.com/gballet/go-verkle/actions/workflows/block_replay.yml/badge.svg)](https://github.com/gballet/go-verkle/actions/workflows/block_replay.yml)
 
+
 # go-verkle
 
-An implementation of [Verkle trees](https://dankradfeist.de/ethereum/2021/06/18/verkle-trie-for-eth1.html). When production-ready, the code is to be merged back into go-ethereum.
+> A Go implementation of Verkle Tree datastructure defined in the [spec](https://github.com/crate-crypto/verkle-trie-ref/tree/master/verkle). 
 
-Node width is set to 256 children.
 
-### Running the tests
+## Test & Benchmarks
 
+To run the tests and benchmarks, run the following commands:
+```bash
+$ go test ./...
 ```
-$ go test .
+
+To run the benchmarks:
+```bash
+go test ./... -bench=. -run=none -benchmem
 ```
 
-### Benchmarks
+## Security
 
-```
-$ go test . -bench Bench
-```
+If you find any security vulnerability, please don't open a GH issue and contact repo owners directly.
```

Comment:
I don't know if we want a reference email which would be way better than making the person figure out who (and how) to contact repo owners.